cookbooks/web/attributes/default.rb

```diff
@@ -1,3 +1,6 @@
 default[:web][:base_directory] = "/srv/www.openstreetmap.org"
 default[:web][:pid_directory] = "/var/run/web"
 default[:web][:log_directory] = "/var/log/web"
+
+# need the APT source for cgimap to be able to install the package
+default[:apt][:sources] = node[:apt][:sources] | ["openstreetmap-cgimap"]
```

Comment:
Can we put this in the recipe please so that only machines using cgimap get it? I think there are other examples where we do that...